### PR TITLE
cors and other updates for websocket ec

### PR DESCRIPTION
**Pull request by wyliekasai**
let me know if there are any changes i should make!

### CODE REVIEW

File: assignments/sa/websockets/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ We're going to build a [websocket](https://en.wikipedia.org/wiki/WebSocket) base
 🚀 Clone your **Lab3 with firebase**.  Create a new branch with `git checkout -b websockets`. We'll need to refactor this very slightly to not use firebase but use our new backend instead, and using a branch seems reasonable for this.
 
 
-🚀 Clone or Fork [express-babel-starter](https://github.com/dartmouth-cs52-19s/express-babel-starter) for our new server component. You can create a new github repo in your own account if you want for this and add it as a remote. 
+🚀 Clone or Fork [express-babel-starter](https://github.com/dartmouth-cs52/express-babel-starter) for our new server component. You can create a new github repo in your own account if you want for this and add it as a remote.
```
Comment (wyliekasai) [inline assignments/sa/websockets/index.md:21]: when i made the server, i found that the most updated version existed at this link

File: assignments/sa/websockets/index.md [hunk 2]
```diff
@@ -206,7 +211,7 @@ io.on('connection', (socket) => {
 ```
 
 
-Ok we have some basic functionality now. Let's work on the frontend for a little bit. We'll come back and add more listeners here. Don't forget to start up mongod and also `npm run dev` the server.
+Ok we have some basic functionality now. Let's work on the frontend for a little bit. We'll come back and add more listeners here. Don't forget to start up mongod and also `npm start` the server.
```
Comment (wyliekasai) [inline assignments/sa/websockets/index.md:214]: 'run dev' didn't work for me so i changed it to start! (but maybe mine was set up incorrectly?)

File: assignments/sa/websockets/index.md (hunk 1 above)
Comment (timofei7) [reply assignments/sa/websockets/index.md:21]: perfect yup - i used to fork them over but then just settled on one

File: assignments/sa/websockets/index.md (hunk 2 above)
Comment (timofei7) [reply assignments/sa/websockets/index.md:214]: this was confusing for people so i made it npm start just like the frontend, good find!

**Review by timofei7 — APPROVED** [review]
love it, thanks!

**Pull request merged**